modules/layers/src/contour-layer/contour-layer.js

```diff
@@ -24,6 +24,7 @@ import {
   _pointToDensityGridData as pointToDensityGridData
 } from '@deck.gl/core';
 import {LineLayer} from '@deck.gl/layers';
+import {_SolidPolygonLayer as SolidPolygonLayer} from '@deck.gl/layers';
```

Comment:
For this line and above: you are importing from the same module, just use relative path??